packages/table/src/common/internal/focusedCellUtils.ts

```diff
@@ -26,13 +27,72 @@ export function getInitialFocusedCell(
         // use the current focused cell from state
         return focusedCellFromState;
     } else if (selectedRegions.length > 0) {
-        // focus the top-left cell of the first selection
+        // focus the top-left cell of the last selection
```

Comment:
note this change. focusing the _last_ selection is more consistent with other focused-cell behavior.

Reply:
Huh, ok. Either way is fine as long as we're consistent.